cmp/ed4980cd-7ae3-46a1-9ac5-4f922be7dc63/component.lp

```diff
@@ -0,0 +1,57 @@
+(librepcb_component ed4980cd-7ae3-46a1-9ac5-4f922be7dc63
+ (name "MAX1698")
+ (description "High-Efficiency Step-Up Current Regulator for LEDs.")
+ (keywords "max1698,max1698a,led")
+ (author "Danilo Bargen")
+ (version "0.1")
+ (created 2018-11-09T19:22:24Z)
+ (deprecated false)
+ (category cda9162b-fa94-4b99-abe0-00fc02c6f5c1)
+ (schematic_only false)
+ (default_value "{{PARTNUMBER or DEVICE}}")
+ (prefix "U")
+ (signal 3e9f34ad-8d24-42f1-93a0-7882ba003cda (name "SHDN") (role passive)
```

Comment:
```suggestion
 (signal 3e9f34ad-8d24-42f1-93a0-7882ba003cda (name "/SHDN") (role passive)
```

Reply:
Should this slash be rendered with an overline? If yes, that doesn't seem to work.

Is the slash some kind of convention? I'd be more used to a tilde or an exclamation mark for inverted logic. KiCad uses the tilde.

Reply:
A slash is commonly used for inverted signals, even in datasheets.

> KiCad uses the tilde.

But the tilde is not rendered, it's to create an overline, isn't it? LibrePCB does not yet support overlines, so IMHO the only commonly known alternative is a slash (or a "N", but sometimes that's confusing).

Reply:
Yes, it creates an overline. You can also terminate a started overline with another tilde.

According to [this link](https://electronics.stackexchange.com/questions/117184/how-to-use-high-and-low-naming-with-negation-on-eagle) Eagle does the same, but with exclamation marks.

So far I think I've only seen overlines or the `N` prefix in datasheets, as far as I can remember.

Do you plan on supporting overlines in the future? Have you already decided on a syntax for that? In that case we could already start using that. (I would definitely not use the slash for that, since it may also be used for separating multiple pin functions.)

Reply:
It's not relevant whether this feature will some time be available in LibrePCB - currently it's not available, so we can't use it ;)

And I think the only notation (beside the overline) which is commonly used, is the starting slash. As an example, I typed "buffer" into the DigiKey search field and opened the datasheet of the first search result (seriously!) -> overline in the symbol, but slash in the truth table:
![grafik](https://user-images.githubusercontent.com/5374821/49186774-cf4d5b80-f365-11e8-9cbc-27c407c8be55.png)